test/StatsTestCases.test.js

```diff
@@ -73,14 +73,20 @@ describe("StatsTestCases", () => {
 				if(!hasColorSetting) {
 					actual = actual
 						.replace(/\u001b\[[0-9;]*m/g, "")
-						.replace(/[0-9]+(\s?ms)/g, "X$1");
+						.replace(/[0-9]+(\s?ms)/g, "X$1")
+						.replace(
+							/[A-z]{3} [A-z]{3} \d\d \d{4} \d\d:\d\d:\d\d GMT[+-]\d{4}( \(\w{3,4}\))?/g,
```

Comment:
looks like this regex isn't quite right - see errors on appveyor: https://ci.appveyor.com/project/sokra/webpack/build/1.0.6407/job/7n2u59afbs3gah5a

Reply:
gonna change "\w{3,4}" to "\w+"